src/Sylius/Bundle/CoreBundle/DataFixtures/ShopConfigurationFixtures.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\CoreBundle\DataFixtures;
+
+use Doctrine\Bundle\FixturesBundle\Fixture;
+use Doctrine\Bundle\FixturesBundle\FixtureGroupInterface;
+use Doctrine\Persistence\ObjectManager;
+
+class ShopConfigurationFixtures extends Fixture implements FixtureGroupInterface
+{
+    public function load(ObjectManager $manager): void
+    {
+    }
+
+    public static function getGroups(): array
+    {
+        return ['shop_configuration'];
```

Comment:
Does it make sense to inject the groups through the constructor?

Reply:
Hum... for now I don't see any reason to change these groups.

Reply:
Either we run all the tests (with all groups) or we can run all groups we want. So I don't see any reason to change these group names. But maybe I am wrong and will se some reasons on customization part. But for now, I suggest to keep it as simple as this.